tasks/couch-security.js

```diff
@@ -22,7 +22,8 @@ module.exports = function(grunt) {
     }
 
     var req = request.defaults(defaults);
-    req(url + '/_security', { body: security }, function(err, resp, data) {
+    var securityJson = JSON.parse(security);
```

Comment:
We could better just use `grunt.file.readJSON` instead of `grunt.file.read` in the task definition.